packages/create-bison-app/template/constants.ts

```diff
@@ -3,3 +3,5 @@ export const EMAIL_REGEX = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
 
 // The key used to store the login token in the auth cookie
 export const LOGIN_TOKEN_KEY = 'myapp-user';
```

Comment:
1. are we using this anywhere? I don't think so...
2. If we are -- Seeing this is a template file, we should place the real app name here (one less thing to change)